src/styles/main.scss

```diff
@@ -1,3 +1,11 @@
-body {
+@import "./utils/variables";
+@import "./utils/mixins";
+@import "./utils/functions";
+
+@import "./blocks/calendar";
+
+* {
   margin: 0;
```

Comment:
Avoid using global wildcard (*) selector for styling styles. Reset margin and padding only for elements where it is necessary
Only one property could be in * selector -` box-sizing: border-box`

Reply:
I understand. But there are so many tasks, where it was asked to reset margins and paddings for all. Every mentor sees it in his own way. What is right?

Reply:
If i'll not reset margins and paddings, my calendar becomes some strange: 

![image](https://user-images.githubusercontent.com/126285904/230724038-85ebd3a7-fd6a-422a-85de-db8e9dd558d7.png)

Reply:
I'll leave margins and paddings in this task in this block

Reply:
I meant to reset margin and padding for specific tags, nor for all selectors.
in your case reset for `body` and `p`